lightning/src/ln/msgs.rs

```diff
@@ -570,6 +570,77 @@ pub struct ChannelUpdate {
 	pub contents: UnsignedChannelUpdate,
 }
 
+/// A query_channel_range message that can be sent or received from a peer
+#[derive(Clone, Debug)]
+pub struct QueryChannelRange {
+	/// The genesis hash of the blockchain being queried
+	pub chain_hash: BlockHash,
+	/// The height of the first block for the channel UTXOs being queried
+	pub first_blocknum: u32,
+	/// The number of blocks to include in the query results
+	pub number_of_blocks: u32,
+}
+
+/// A reply_channel_range message that can be sent or received from a peer.
```

Comment:
Every p2p messages can be sent or received, that doesn't add that much information. I realized we do this for every other message, you can drop it IMO.